qfieldsync/gui/package_dialog.py

```diff
@@ -140,20 +136,51 @@ def setup_gui(self):
         else:
             self.show_package_page()
 
-    def get_export_folder_from_dialog(self):
+    def get_export_folder(self):
         """Get the export folder according to the inputs in the selected"""
-        # manual
-        return self.manualDir.text()
+        export_dirname = self.qfield_preferences.value("exportDirectoryProject")
+        if not export_dirname:
+            export_dirname = os.path.join(
+                self.qfield_preferences.value("exportDirectory"),
+                fileparts(QgsProject.instance().fileName())[1],
+            )
+        return QDir.toNativeSeparators(str(export_dirname))
+
+    def get_export_filename_suggestion(self):
```

Comment:
```suggestion
    def get_export_filename_suggestion(self) -> str:
```

Please add typing on all new methods you add.